Src/BootCamp.Chapter/Program.cs

```diff
@@ -9,6 +9,63 @@ class Program
     {
         static void Main(string[] args)
         {
+            PersonOneBMI();
```

Comment:
Do you need two functions :) ?
Two calls- yes, but the same function can be reused!